### PR TITLE
Laravel 5.8 Upgrade

### PR DESCRIPTION
**Pull request by atoff**
Important Deployment Note:
 - The VATSIM SSO Private Key in the .env must now be in the normal format (i.e without literal `\n`)

Closes CORE-357

### CODE REVIEW

File: app/Http/Controllers/Adm/Dashboard.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function getIndex()
             return $statistics;
         });
 
-        $membershipStats = Cache::remember('statistics.membership.graph', 60 * 24, function () {
+        $membershipStats = Cache::remember('statistics.membership.graph', 60 * 24 * 60, function () {
```
Comment (AxonC) [inline app/Http/Controllers/Adm/Dashboard.php]:
```suggestion
        $membershipStats = Cache::remember('statistics.membership.graph', (60 * 24) * 60, function () {
```

File: app/Http/Controllers/Adm/VisitTransfer/Dashboard.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ public function getDashboard()
             return $statistics;
         });
 
-        $statisticsGraph = Cache::remember('visittransfer::statistics.graph', 60 * 24, function () {
+        $statisticsGraph = Cache::remember('visittransfer::statistics.graph', 60 * 24 * 60, function () {
```
Comment (AxonC) [inline app/Http/Controllers/Adm/VisitTransfer/Dashboard.php]:
```suggestion
        $statisticsGraph = Cache::remember('visittransfer::statistics.graph', (60 * 24) * 60, function () {
```

File: app/Http/Controllers/NetworkData/Dashboard.php [hunk 3]
```diff
@@ -20,7 +20,7 @@ public function getDashboard()
             return $statistics;
         });
 
-        $statisticsGraph = Cache::remember('networkdata::statistics.graph', 60 * 24, function () {
+        $statisticsGraph = Cache::remember('networkdata::statistics.graph', 60 * 24 * 60, function () {
```
Comment (AxonC) [inline app/Http/Controllers/NetworkData/Dashboard.php]:
```suggestion
        $statisticsGraph = Cache::remember('networkdata::statistics.graph', (60 * 24) * 60, function () {
```

**Comment by CalumTowers** [comment]
@atoff Before merging, it appears to be encountering a few issues deploying on Beta. Down to an NPM issue. Can you check Deploybot and review the error log please?

@daviesnathan FYI, could be a versioning issue.

**Comment by atoff** [comment]
@CalumTowers I am unable to log into deploybot to review this, but I wouldn't have though it a problem with the code itself.

**Pull request merged**